_episodes/03-file-transfer.md

```diff
@@ -1,43 +1,48 @@
-
 ---
 title: "Transferring Files"
-teaching: 10
-exercises: 0
+teaching: 15
+exercises: 10
 questions:
-- "How to use wget, curl and lftp to transfer file?"
+- "How to use Wget, curl and rsync to transfer file?"
 objectives:
-- "FIXME"
+- "Understand that Wget, curl and rsync can transfer files"
 keypoints:
-- "FIXME"
+- "Wget is a utility for downloading remote webpages.
+It includes options to mirror an entire site."
+- "curl is another utility for downloading remote webpages.
+It defaults to outputting the result on screen, this can be piped to other
+programs."
+- "rsync is a utility for transferring files. It can use the SSH protocol and
+is useful for mirroring complicated directory structures from one computer to
+another."
```

Comment:
```suggestion
- "Wget is a utility for downloading remote webpages.
  It includes options to mirror an entire site."
- "curl is another utility for downloading remote webpages.
  It defaults to outputting the result on screen, this can be piped to other
  programs."
- "rsync is a utility for transferring files. It can use the SSH protocol and
  is useful for mirroring complicated directory structures from one computer to
  another."
```